DESCRIPTION.md

```diff
@@ -0,0 +1,734 @@
+# Introduction
+
+Welcome to bunq!
+
+- The bunq API is organised around REST. JSON will be returned in almost all responses from the API, including errors but excluding binary (image) files.
+- All calls made through bunq Doc are executed on a sandbox environment. No real money is used and no transactions to external bank accounts can be done.
+- Please configure your implementation to send it's API requests to https://sandbox.public.api.bunq.com/v1.
+- There is an Android client available of our app that connects to the bunq Sandbox environment.
+- If you have a developer key, please proceed to the Settings page and enter this key to enable execution of test calls.
+
+## Get Started
+
+1. Create a user account with your phone. Afterwards, you can use this account to create an API key from which you can make API calls. You can find API key management under 'Profile' -\> 'Security'.
+2. Register a device. A device can be a phone (private), computer or a server (public). You can register a new device by using the installation and device-server calls.
+3. Open a session. Sessions are temporary and expire after the same amount of time you have set for auto logout in your user account.
+4. Make your first call!
+
+## Versioning
+
+Our API is currently in an initial testing phase. This means we will iterate quickly to improve it and related tooling. This also allows us to quickly process your feedback (which we are happy to receive!). Therefore, we have chosen not to attach any version numbers to the changes just yet. We will inform you in a timely manner of any important changes we make before they are deployed on together.bunq.com.
+
+Once the speed of iteration slows down and more developers start using the API and its sandbox we will start versioning the API using the version number in the HTTP URLs (i.e. the '/v1' part of the path). We will inform you when this happens.
+
+# Authentication
+
+- We use encryption for all API calls. This means that all requests must use HTTPS. The HTTP standard calls will fail. You should also use SSL Certificate Pinning and Hostname Verification to ensure a secure connection with bunq.
+- In order to make API calls you need to register a device and open a session.
+- We use RSA Keys for signatures headers and encryption.
+- API calls must contain a valid authentication token in the headers.
+- The auto logout time that you've set for your user account is also effective for your sessions. If a request is made 30 minutes before a session expires, the session will automatically be extended.
+## Device Registration
```

Comment:
I believe the new strict Markdown requires enters around headers.
Can you doublecheck?